packages/base/src/css/SystemCSSVars.css

```diff
@@ -15,3 +16,9 @@
 [dir="ltr"] {
     --_ui5_dir:ltr;
 }
+
+.sapUiNativeScrollbars,
```

Comment:
Probably you modelled this based on content density. There we have 3 ways of applying it because of the OpenUI5 compatibility (the classes already exist and somebody might have set them on the body). However, since this is new development, we can have only 1 or 2 ways of doing it.

I checked what we documented for Content Density:
https://github.com/SAP/ui5-webcomponents/blob/main/docs/2-advanced/02-RTL-and-compact-mode.md#compact-mode
and it's just the data attribute and the new class (not the old class). Therefore, `. sapUiNativeScrollbars` is probably not necessary as it doesn't have a "historic equivalent" from the OpenUI5 world.

Reply:
Ignore this comment, as discussed